lib/rouge/lexers/r.rb

```diff
@@ -66,7 +66,7 @@ def self.detect?(text)
         # function call, i.e. followed by an opening parenthesis.
         # `Name::Builtin` would be more logical, but is usually not
         # highlighted specifically; thus use `Name::Function`.
-        rule %r/\b(?<!.)(#{PRIMITIVE_FUNCTIONS.join('|')})(?=\()/, Name::Function
+        rule %r/(?<![.])(#{PRIMITIVE_FUNCTIONS.join('|')})(?=\()/, Name::Function
```

Comment:
Negative lookbehind was always broken until now. Turns out `.` matches anything, and we actually cared about a literal period.